statechart/models/statechart_mixin.py

```diff
@@ -40,6 +40,7 @@ class StatechartMixin(models.AbstractModel):
 
     _name = 'statechart.mixin'
     _description = 'Statechart Mixin'
+    _sc_has_allowed_events = False
```

Comment:
```suggestion
    _sc_has_allowed_events_enabled = False
```

Reply:
BTW, why do we need this flag ? If it is disabled I guess the feature is not exposed to users anyway ?
And if it is, is it not better to say `sc_has_allowed_events=True` ?

Reply:
yes but if it's not needed, I suppose nobody reads or search on the field so it has no impact, correct ?

Reply:
In other words, what is the drawback if we remove `_sc_has_allowed_events_enabled` ?

Reply:
it's more a "nice to have", by default the field is disabled, so the compute is not triggered for each `sc_state` change and prevent any performance impact if users use it in search.